tests/assertions/internal-libraries.n

```diff
@@ -52,7 +52,7 @@ let pub main = [_: ()] -> cmd[()] {
 		response: str
 		return: json.value
 	}]
-	assert value request.get("https://httpbin.org/base64/SFRUUEJJTiBpcyBhd2Vzb21l", mapFrom([]))! = {
+	assert value request.get("https://httpbin.org/base64/SFRUUEJJTiBpcyBhd2Vzb21l", mapFrom([("", "")]))! = {
```

Comment:
I feel like this is a hack and an issue on the Python implementation's side, not with the test

Reply:
Ok, I'll open an issue